bin/pdc_client

```diff
@@ -163,7 +166,12 @@ if __name__ == "__main__":
                 data[key] = ''
 
     try:
-        client = PDCClient(options.server)
+        ssl_verify = args.ca_cert if args.ca_cert else not args.insecure
```

Comment:
ca_cert or insecure should be in options, not args which is different with argparse